### PR TITLE
Nav Bar changes

### PR DESCRIPTION
**Pull request by yumnali**
Completed remaining features of the nav bar

### CODE REVIEW

File: public/imgs/sms-8-svgrepo-com.svg
Comment (asimkhan73301) [inline public/imgs/sms-8-svgrepo-com.svg:1]: Can we change this icon to the following?

https://www.svgrepo.com/svg/522070/comment-3

**Review by asimkhan73301 — CHANGES_REQUESTED** [review]
Great work! Let's add the minor changes requested.

File: public/imgs/sms-8-svgrepo-com.svg
Comment (yumnali) [reply public/imgs/sms-8-svgrepo-com.svg:1]: Please change yourself

**Review by asimkhan73301 — APPROVED** [review]
LGTM 👍

**Pull request merged**